aten/src/ATen/native/Sorting.cpp

```diff
@@ -226,9 +226,19 @@ Tensor quantile_compute(
   // NOTE: this check is only performed when running on the CPU to avoid
   // synchronizing an accelerator with the CPU
   if (self.device().is_cpu()) {
-    TORCH_CHECK(
-        q.ge(0).logical_and_(q.le(1)).all().item<bool>(),
+    auto q_in_range = q.ge(0).logical_and_(q.le(1)).all();
+    // For Composite Compliance, we use `nonzero().numel()`
```

Comment:
@zou3519 what do you think of this approach? This works but won't work with `BatchedTensor` due to dynamic shape.

Reply:
On second thought, I guess this is not good as this would unnecessarily limit `vmap` support in these cases.

Reply:
Yeah ideally we would do this without nonzero(), nonzero() is also a bit troubling for things like dynamic shape support (nonzero can return a tensor with arbitrary shape)